addons/supplies/XEH_postInitServer.sqf

```diff
@@ -0,0 +1,19 @@
+#include "script_component.hpp"
+
+GVAR(playerTraits) = createHashMap;
+
+[QGVAR(storeTraits), {
+    GVAR(playerTraits) set _this;
+}] call CBA_fnc_addEventHandler;
+
+[QGVAR(loadTraits), {
```

Comment:
I suggest `params ["_player"];` to make it unambiguous as to what object we are dealing with.